spring-integration-mail/src/main/resources/org/springframework/integration/mail/config/spring-integration-mail-4.3.xsd

```diff
@@ -363,4 +365,37 @@
 		</xsd:attribute>
 	</xsd:complexType>
 
+	<xsd:attributeGroup name="inboundMappingGroup">
+		<xsd:attribute name="header-mapper" type="xsd:string">
+			<xsd:annotation>
+				<xsd:documentation><![CDATA[
+	A reference to a `org.springframework.integration.mapping.HeaderMapper<MimeMessage>` bean.
+	When not supplied, the message payload will be the raw MimeMessage with no header mapping.
+	When a mapper is provided, the mail headers are mapped to 'MessageHeaders' and the payload
+	will depend on the email contents, and the setting of 'multipart-as-bytes'. The framework
+	provides a `DefaultMailHeaderMappe`; see the reference manual regarding the headers mapped
```

Comment:
Typo: DefaultMailHeaderMappe**r**
Also seems for me tend to use "'" instead of "`" in the XSD docs.